Makefile

```diff
@@ -15,7 +15,9 @@ UNAME := $(uname)
 
 setup:
 	- @if [ -z "${HELM_INSTALLED}" ] && [[ "${UNAME}" == 'Darwin' ]]; then \
-			brew install kubernetes-helm ; \
+			brew install helm ; \
```

Comment:
Seems like this spacing is tabs? I think we are supposed to use spaces :)

Reply:
It's a Makefile. It has to be tabs 

![tabs and spaces](https://media2.giphy.com/media/LROKNqr26NG48/giphy.gif)